KubeArmor/core/crioHandler.go

```diff
@@ -48,36 +44,11 @@ type CrioContainerInfo struct {
 // Crio Handler
 var Crio *CrioHandler
 
-// init Function
-func init() {
-	Crio = NewCrioHandler()
```

Comment:
Had to move this out of the `init` function and into `MonitorCrioEvents` because init function is called before config is read which leads to `CRI_SOCKET` and `criSocket` flag not having any effect.